src/cpp/fastdds/subscriber/DataReaderImpl.hpp

```diff
@@ -139,28 +138,24 @@ class DataReaderImpl
      */
     TypeSupport type();
 
+    /**
+     * Get TopicDescription
+     * @return TopicDescription
+     */
+    const TopicDescription* topic() const;
```

Comment:
```suggestion
    const TopicDescription* get_topic_description() const;
```

Reply:
I will use `get_topicdescription`, as this is the method name on the standard